index.html

```diff
@@ -565,30 +537,26 @@
           </li>
           <li data-link-for="PaymentDetailsInit">Process the total:
             <ol>
-              <li>If <var>details</var>.<a>total</a>.<a data-lt=
-              "PaymentItem.amount">amount</a>.<a data-lt=
-              "PaymentCurrencyAmount.value">value</a> is not a <a>valid decimal
-              monetary value</a>, then <a>throw</a> a <a>TypeError</a>;
-              optionally informing the developer that the value is invalid.
-              </li>
-              <li>If the first <a>code point</a> of
+              <li>Let <var>total</var> be
               <var>details</var>.<a>total</a>.<a data-lt=
-              "PaymentItem.amount">amount</a>.<a data-lt=
-              "PaymentCurrencyAmount.value">value</a> is U+002D HYPHEN-MINUS,
-              then <a>throw</a> a <a>TypeError</a>, optionally informing the
-              developer that the total can't be negative.
+              "PaymentItem.amount">amount</a>.
+              </li>
+              <li>
+                <a>Check and canonicalize <var>total</var></a>. Rethrow any
+                exceptions.
```

Comment:
In theory rethrowing exceptions is not necessary (https://infra.spec.whatwg.org/#algorithm-control-flow), but in practice we should stay consistent, and I believe the spec mostly includes this verbiage right now. So, just noting FYI.